l10n_it_fatturapa_out_di/tests/fatturapa_common.py

```diff
@@ -0,0 +1,27 @@
+from odoo.addons.l10n_it_fatturapa_out.tests.fatturapa_common import (
+    FatturaPACommon as _FatturaPACommon,
+)
+
+
+class FatturaPACommon(_FatturaPACommon):
+    def setUp(self):
+        super().setUp()
+
+        def _fix_payability(tax, ref):
+            reftax = self.env.ref(ref).sudo()
+            tax.payability = reftax.payability
+
+        # XXX - to be moved to l10n_it_fatturapa_out.tests.fatturapa_common ?
+        _fix_payability(self.tax_22, "l10n_it_fatturapa.tax_22")
+        _fix_payability(self.tax_10, "l10n_it_fatturapa.tax_10")
+        _fix_payability(self.tax_22_SP, "l10n_it_fatturapa.tax_22_SP")
+
+    def getAttachment(self, name, module_name=None):
+        if module_name is None:
```

Comment:
Che succede se, per qualsiasi motivo, `module_name` è `False` o `''`? Credo che un semplice `if not module_name` copra maggiori casi di nullable.

Reply:
Premesso che per #2507 questo file dovrà sparire (nel senso che va tutto spostato, i moduli non devono avere la loro copia locale), questo codice è quello usato sin dalla 12.

https://github.com/OCA/l10n-italy/blob/c366105274f26c5636d217180e603fcca8903bcf/l10n_it_fatturapa_out/tests/fatturapa_common.py#L135-L140

Io sono perfettamente d'accordo con la tua ossevazione ma non credo questo sia il posto per discuterne, cambiare il comportamento solo in un modulo ponte lasciando tutto il resto inalteranto ha poco senso. Paradossalmente, meglio lasciarlo in linea con lo standard (per quanto discutibile) piuttosto che creare un'eccezione qui.

Reply:
La tua osservazione mi sembra ragionevolmente corretta. Allora ritiro la segnalazione.